sematic/plugins/abstract_kuberay_wrapper.py

```diff
@@ -65,8 +70,10 @@ class ScalingGroup:
     max_workers: int = 1
 
     def __post_init__(self):
-        if self.min_workers <= 0:
-            raise ValueError("min_workers must be >= 1")
+        if self.min_workers < 0:
+            raise ValueError("min_workers must be >= 0")
```

Comment:
Does it ever make sense to allow for 0 workers?

Reply:
Yes; Ray can operate with only a head node (the head can also execute tasks).